### ISSUE TITLE
Link has expired!

### ISSUE THREAD

**Issue by GewelsJI** [pytorch/pytorch.github.io#781]
## 🕋 Website
 This link may be expired:
https://pytorch.org/docs/_modules/torch/nn/modules/batchnorm.html#BatchNorm2d
